Source/Python/WarpXWrappers.cpp

```diff
@@ -710,9 +710,10 @@ namespace
     }
     void warpx_SyncCurrent (
```

Comment:
There is a symbol error left `undefined symbol: warpx_SyncCurrent` can you update `Source/Python/WarpXWrappers.H` as well? :)